src/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,62 @@
+import clsx from "clsx";
+import React, { useState } from "react";
+
+import { Button } from "../button";
+import { Toolbar, ToolbarProps } from "../toolbar";
+import { P } from "../typography";
+import "./tabs.scss";
+
+export type TabProps = {
+  label: string;
+  content: React.ReactNode;
+};
+
+export type TabsProps = React.PropsWithChildren<
+  {
+    tabs: TabProps[];
+    onTabChange?: (activeTabIndex: number) => void;
+  } & ToolbarProps
+>;
+
+/**
+ * A tabs component meant to allow for cycling through different content sections. Allows for passing custom react nodes as content for each tab.
+ * @param tabs
+ * @param onTabChange
+ * @param props
+ * @constructor
+ */
+export const Tabs: React.FC<TabsProps> = ({
+  onTabChange,
+  tabs,
+  className,
+  ...props
+}) => {
+  const [activeTab, setActiveTab] = useState(0);
+
+  const handleTabClick = (index: number) => {
+    setActiveTab(index);
+    onTabChange && onTabChange(index);
+  };
+
+  return (
+    <Toolbar className={clsx("mykn-tabs", className)} {...props}>
+      <div className="mykn-tabs__buttons">
+        {tabs.map((tab, index) => (
+          <React.Fragment key={index}>
```

Comment:
FYI: I usually use the shorthand syntax (`<></>`) https://react.dev/reference/react/Fragment

Reply:
Yeah I usually use the shorthand aswell, but in this case we need to pass it the `key` attribute, which is not possible to be passed in the shorthand.